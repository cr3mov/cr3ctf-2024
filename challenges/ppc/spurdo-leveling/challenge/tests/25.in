147820265408139069 994545160357211723
1 5899815702860
1 70305532695447
