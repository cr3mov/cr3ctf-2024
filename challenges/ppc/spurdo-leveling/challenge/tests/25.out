11112 11112 11112
